src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -42,7 +42,7 @@ private int Run(string[] args)
                     string arg = args[i];
                     string value = string.Empty;
 
-                    if (arg.StartsWith("/") || arg.StartsWith("-"))
+                    if (arg.StartsWith("-"))
                     {
                         int colonPos = arg.IndexOf(":");
```

Comment:
Think it a bit more. We should follow the convention from the beginning; otherwise, it would be more painful to change in the future.

We should support "--assembly foo.dll" instead of "--assembly:foo.dll".

Reply:
@zhenlan Thanks for your comments. What's the concern about using colon (--assembly:foo.dll) here?

Reply:
Not a concern but more of a convention in dotnet CLI. If we decide to follow the convention in the future, it will be a breaking change.